src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -480,7 +486,7 @@ public CloudSdk build() {
      * @throws AppEngineException if not found
      */
     @Nonnull
-    private Path discoverSdkPath() {
+    private Path discoverSdkPath() throws AppEngineException {
```

Comment:
Not sure if this is outside the scope of this PR, but it would be nice if there were a more specific checked exception we could use here - something like `CloudSdkNotFoundException`